supertokens_python/logger.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import json
+import logging
+from datetime import datetime, timezone
+from os import getenv
+from typing import TextIO, Union
+
+from .constants import VERSION
+
+# Configure logger
+logger = logging.getLogger()
+log_level_str = getenv('LOG_LEVEL', "").lower()
+default_log_level = logging.INFO
+log_level_dict = {"debug": logging.DEBUG, "info": logging.INFO}
+logger.setLevel(log_level_dict.get(log_level_str, default_log_level))
+
+
+def _get_log_timestamp():
+    return datetime.now(timezone.utc).isoformat()
+
+
+class CustomStreamHandler(logging.StreamHandler):  # type: ignore
```

Comment:
Why does this have `# type: ignore`?

Reply:
> I tried it last time but I think my python extension didn't respond fast enough and that made me believe that this doesn't work.

I think it was the same issue here as well. I was also confused about why it's asking for a type. Thanks again! :)

Fixed it.

Reply:
Adding `# pyright: reportUnknownMemberType=false` to the top of the file is not a good idea. It's worse than keeping that one line to have `# type: ignore`. It's worse cause for other areas this error might get suppressed.